sdk/metric/correct_test.go

```diff
@@ -289,3 +296,45 @@ func TestDefaultLabelEncoder(t *testing.T) {
 	})
 	require.Equal(t, "I=1,U=1,I32=1,U32=1,I64=1,U64=1,F64=1,F64=1,S=1,B=true", encoded)
 }
+
+func TestObserverCollection(t *testing.T) {
+	ctx := context.Background()
+	batcher := &correctnessBatcher{
+		t: t,
+	}
+	sdk := sdk.New(batcher, sdk.NewDefaultLabelEncoder())
+	meter := metric.WrapMeterImpl(sdk)
+
+	_ = Must(meter).RegisterFloat64Observer("float.observer", func(result metric.Float64ObserverResult) {
+		// TODO: The spec says the last-value wins in observer
```

Comment:
That could be my misunderstanding of what last-value aggregation within a labelset and measure aggregation across labelsets meant in the spec. Maybe file an issue about it and assign it to me?

Reply:
I had forgotten which details are in the spec and which are in the OTEP that's not yet merged. This detail is not yet merged, so there's not really a problem. I filed a ticket #564.